man/xml/daps.xml

```diff
@@ -547,15 +547,15 @@
    <title>Other Commands</title>
    <variablelist>
     <varlistentry>
-     <term><option>checklink</option></term>
+     <term><option>getimages</option></term>
      <listitem>
-      <para>Linkchecker for links from <![CDATA[<ulink>]]> tags.</para>
+      <para>List or display images from a profiled DocBook XML file.</para>
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term><option>getimages</option></term>
+     <term><option>linkcheck</option></term>
      <listitem>
-      <para>List or display images from a profiled DocBook XML file.</para>
+      <para>Linkchecker for links from <![CDATA[<ulink>]]> tags.</para>
```

Comment:
* this tag name is wrong for DocBook 5
* sgmltag (in DB4) and tag (in DB5) will generate the correct tag, with only the tiniest bit more markup.

```suggestion
      <para>Linkchecker for URLs within <sgmltag class="starttag">link</sgmltag> and <sgmltag class="starttag">link</sgmltag> tags.</para>
```